os/board/rtl8721csm/src/component/common/drivers/wlan/realtek/include/autoconf.h

```diff
@@ -507,7 +507,7 @@ extern unsigned int g_ap_sta_num;
 		#undef RX_AGGREGATION
 		#define RX_AGGREGATION 1
 		#undef NOT_SUPPORT_40M
-		#define RX_SHORTCUT 1
+		//#define RX_SHORTCUT 1

```

Comment:
@gexuyan How large packet do you want to segment? Why does this forced segmentation help tcp to avoid crash?

Reply:
hi @davidfather , the tcp crash is due to ip layer receive 60k bytes size packets and the bulky memory allocated for large packets will touch secure memory region, which will raise security hardfault and leads to crash. Originally, received packets will go through shortcut path which doesn't include segmentation procedure, after closing RX_SHORTCUT, received large packets will be segmented then don't require a continuous bulky memory.